libs/wingc/src/type_check.rs

```diff
@@ -1213,6 +1213,7 @@ impl<'a> TypeChecker<'a> {
 				Literal::Boolean(_) => self.types.bool(),
 			},
 			ExprKind::Binary { op, left, right } => {
+				panic!("TEST");
```

Comment:
:point_up: probably need to clean this up, but maybe we do want some debug capability to cause the compiler to panic at a certain point in the code. I propose the panic emoji (😱) and adding it to the grammar ;).

Reply:
ok I'm on it as part of my pr